core/tests/webdriverio_desktop/contributorDashboard.js

```diff
@@ -183,12 +183,12 @@ describe('Contributor dashboard page', function() {
       await action.click('Basic Settings', basicSettings);
     }
     await explorationEditorSettingsTab.setTitle('exp1');
+    await explorationEditorSettingsTab.setCategory('Algebra');
```

Comment:
Debugging doc: https://docs.google.com/document/d/1TMZkHzNFb-Ua_9Y4CQrm630VqhPawtMYDzOZjZW80e4/edit?usp=sharing

Reply:
This also seems like a FE bug in that the editor shouldn't be loading until the language list is available. Do you agree? If so, could you file a bug with LaCE quality?